src/teacher/Home.js

```diff
@@ -22,15 +20,14 @@ function Home({ api }) {
         <sc.TopTabs>
           <h1>{strings.myClasses}</h1>
         </sc.TopTabs>
-
-        {isLoadingCohorts ? (
-          <LoadingAnimation />
-        ) : (
+        {cohorts ? (
```

Comment:
so this relies on the fact that useState() will initialize the cohorts with null? and then even if there are no cohorts, the variable will now become an empty list? 

I like this pattern more than the one with isLoadingCohorts! Cool job!